lua/telekasten.lua

```diff
@@ -1981,6 +1996,7 @@ local function on_create_with_template(opts, title)
             map("n", "<c-i>", picker_actions.paste_link(opts))
             return true
         end,
+        sort = M.Cfg.sort,
```

Comment:
Is it really necessary here? This picker is for template selection, so it does not really make sense to have it by modified (imo).

Reply:
deleted